app/src/main/java/com/android/sample/ui/utils/RecipeList.kt

```diff
@@ -160,15 +150,15 @@ private fun RecipeCard(
                     horizontalArrangement = Arrangement.SpaceBetween) {
                       RecipeCategories(recipe)
 
-                      RecipePrice(cost = 3, recipe = recipe)
+                      RecipePrice(cost = 3)
                     }
               }
         }
   }
 }
 
 @Composable
-private fun RecipePrice(maxDollars: Int = 3, cost: Int, recipe: Recipe) {
+private fun RecipePrice(maxDollars: Int = 3, cost: Int) {
```

Comment:
Would it be interesting to plainly get rid of this function? As we don't want to use it.

Reply:
Since the primary focus of this PR is the fridge UI, we could address this in a separate PR, along with other small updates, to keep things more organized.

Reply:
Seems right